### ISSUE TITLE
Which pypi repo to use?

### ISSUE THREAD

**Issue by npezolano** [jjstickel/scikit-datasmooth#3]
Looks like we have two:

https://pypi.python.org/pypi/scikit.datasmooth

https://pypi.python.org/pypi/scikits.datasmooth

**Comment by jjstickel** [comment]
Right, I originally used the "scikits" designation, but other active projects are instead using "scikit", and so I changed it. I guess I didn't quite get it right, though, because those projects are "scikit-[name]", while this one is use "scikit.datasmooth" (dot vs dash). Also, the "scikit-" projects do not use the scikits namespace. I'd be happy to be responsive to your pull requests if you want to clean it up! Then maybe I can try to delete (or hide?) the old version-named datasmooth projects on pypi.
